.github/workflows/main.yml

```diff
@@ -31,9 +31,9 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
       - name: Install poetry
-        uses: abatilo/actions-poetry@v3
-        with:
-          poetry-version: ${{ matrix.poetry-version }}
+        shell: bash
+        run: |
```

Comment:
Why Use |?

    Clarity: Even for a single command, using | makes it clear that the value is a multi-line string (and it’s a common convention in GitHub Actions).

    Flexibility: If you later add more commands, you can do so without changing the YAML structure. For example: